soroban-env-host/src/test/stellar_asset_contract.rs

```diff
@@ -1624,13 +1624,13 @@ fn test_account_invoker_auth_with_issuer_admin() {
     test.run_from_account(admin_acc.clone(), || {
         contract.mint(
             &TestSigner::AccountInvoker(admin_acc.clone()),
-            admin_address.clone(),
+            user_address.clone(),
```

Comment:
Could you actually add a test where we mint/burn issuer balance to make sure an error is returned?

Reply:
I was going to add those tests in core to save time (I already added one for burn that will fail when we update the env ref). Do you still want me to add one here?

Reply:
Tests in host are somewhat more concise/easier to digest (and also easier to observe now), but up to you really. It doesn't seem like core tests would provide more coverage for this particular scenario (I just happened to think about this during the core review...)

Reply:
Makes sense. Updated